stacks-signer/src/chainstate.rs

```diff
@@ -0,0 +1,375 @@
+use blockstack_lib::chainstate::nakamoto::NakamotoBlock;
```

Comment:
Missing copyright block

Reply:
Added in fdcaea8b91972ad721d9d9e0579ac89fb531ddc0 (stackslib/src/net/api/tests/mod.rs)